nucliadb_node/src/env.rs

```diff
@@ -0,0 +1,337 @@
+                "WRITER_LISTEN_ADDRESS not defined. Defaulting to: {}",
+                default
+            );
+            default
+        }
+    }
+}
+
+pub fn lazy_loading() -> bool {
+    let default = true;
+    match env::var("LAZY_LOADING") {
+        Ok(var) => match var.parse() {
+            Ok(value) => value,
+            Err(_) => {
+                error!("Error parsing environment variable LAZY_LOADING");
+                default
+            }
+        },
+        Err(_) => {
+            warn!("LAZY_LOADING not defined. Defaulting to: {}", default);
+            default
+        }
+    }
+}
+
+pub fn host_key_path() -> String {
+    let default = String::from("host_key");
+    match env::var("HOST_KEY_PATH") {
+        Ok(var) => match var.parse() {
+            Ok(value) => value,
+            Err(_) => {
+                error!("Error parsing environment variable HOST_KEY_PATH");
+                default
+            }
+        },
+        Err(_) => default,
+    }
+}
+
+pub fn chitchat_port() -> u16 {
+    let default: u16 = 40100;
+    match env::var("CHITCHAT_PORT") {
+        Ok(var) => u16::from_str(&var).unwrap_or_else(|e| {
+            error!("Can't parse CHITCHAT_PORT variable: {e}");
+            default
+        }),
+        Err(_) => {
+            warn!("CHITCHAT_PORT not defined. Defaulting to: {}", default);
+            default
+        }
+    }
+}
+
+pub async fn public_ip() -> IpAddr {
+    let default = IpAddr::from_str("::1").unwrap();
+    match env::var("HOSTNAME") {
+        Ok(v) => {
+            let host = format!("{}:4444", &v);
+            reliable_lookup_host(&host).await
+        }
+        Err(e) => {
+            error!("HOSTNAME node defined. Defaulting to: {default}. Error details: {e}");
+            default
+        }
+    }
+}
+
+pub fn seed_nodes() -> Vec<String> {
+    let default = vec![];
+    match env::var("SEED_NODES") {
+        Ok(v) => v.split(';').map(|addr| addr.to_string()).collect(),
+        Err(e) => {
+            error!("Error parsing environment varialbe SEED_NODES: {e}");
+            default
+        }
+    }
+}
+
+pub fn sentry_url() -> String {
+    let default = String::from("");
+    match env::var("SENTRY_URL") {
+        Ok(var) => match var.parse() {
+            Ok(value) => value,
+            Err(_) => {
+                error!("Error parsing environment variable SENTRY_URL");
+                default
+            }
+        },
+        Err(_) => default,
+    }
+}
+
+pub fn log_level() -> Vec<(String, Level)> {
+    let default = "nucliadb_node=WARN,nucliadb_cluster=WARN,nucliadb_cluster=WARN".to_string();
+    match env::var("RUST_LOG") {
+        Ok(levels) => parse_log_level(&levels),
+        Err(_) => {
+            error!("RUST_LOG not defined. Defaulting to {default}");
+            parse_log_level(&default)
+        }
+    }
+}
+
+pub fn get_sentry_env() -> &'static str {
+    let default = SENTRY_DEV;
+    match env::var("RUNNING_ENVIRONMENT") {
```

Comment:
Should be configured via ENV Var

Reply:
Isn't it? through `RUNNING_ENVIRONMENT`, again I am a bit lost with the logic here because it was not implemented by me. But I will be happy to fix it, just need more pointers about what is wrong.

Reply:
https://github.com/nuclia/nucliadb/blob/relations_on_bm/charts/nucliadb_node/templates/node.cm.yaml#L20 There is a env var that shows the SENTRY_URL, no need to define STAGE and PROD

Reply:
Fixed